gui/common/freenasldap.py

```diff
@@ -1148,8 +1148,8 @@ def get_SRV_records(host):
         )
 
         r = resolver.Resolver()
-        r.timeout = _fs().directoryservice.activedirectory.dns.timeout
         r.lifetime = _fs().directoryservice.activedirectory.dns.lifetime
+        r.timeout = r.lifetime / 3
 
```

Comment:
I understand the reasons you made it this way and that you don't want to mess with sysctl() updates, but I'd keep ability to modify timeout through the sysctl(). How about:

```r.timeout = _fs().directoryservice.activedirectory.dns.timeout / 2```
Or 
```r.lifetime = _fs().directoryservice.activedirectory.dns.lifetime * 2```